packages/cosmwasm-launchpad/src/signingcosmwasmclient.ts

```diff
@@ -149,7 +149,7 @@ export interface PrivateSigningCosmWasmClient {
 }
 
 export class SigningCosmWasmClient extends CosmWasmClient {
-  public readonly senderAddress: string;
+  public readonly signerAddress: string;
```

Comment:
@willclarktech ah right, I did not notice this is public. Will add a CHANGELOG entry for this change.